### PR TITLE
(fix) [SD-3511] Changes in an open item are not saved after logout/login

### PR DESCRIPTION
**Pull request by sivakuna-aap**

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -162,7 +162,9 @@
             this.stop(item);
 
             if (angular.isDefined(item._autosave) && item._autosave !== null) {
-                api(RESOURCE).remove(item._autosave);
+                api(RESOURCE).remove(item._autosave).then(function () {
+                    //authoringWorkspace.removeAutosave(_autosave);
```
Comment (mdhaman) [inline client/app/scripts/superdesk-authoring/authoring.js]: is this required?

Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/authoring.js]: No, forgot to remove. Will remove it.

**Comment by mdhaman** [comment]
:+1: except for the minor comment

**Comment by ioanpocol** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**